paddle/fluid/operators/reduce_ops/reduce_op.h

```diff
@@ -590,7 +581,6 @@ class ReduceGradOp : public framework::OperatorWithKernel {
     }
 #endif
 
-    int in_dtype = ctx.Attr<int>("in_dtype");
     if (in_dtype >= 0) {
```

Comment:
I think the if block (584-588)  is not needed anymore because the condition is already checked above (563-566).

Reply:
You're right, I'll change that. Thank you